src/bootstrap/bin/rustc.rs

```diff
@@ -125,6 +125,11 @@ fn main() {
             cmd.arg("-C").arg(format!("codegen-units={}", s));
         }
 
+        // Emit save-analysis info.
+        if env::var("RUSTC_SAVE_ANALYSIS") == Ok("api".to_string()) {
+            cmd.arg("-Zsave-analysis-api");
+        }
```

Comment:
Oh nvm I see what's going on. bootstrap is using this to communicate with the shim.